src/main/java/de/orchound/gameoflife/model/Board.java

```diff
@@ -12,10 +12,12 @@
 	private final boolean[] initial;
 	public boolean[] target;
 	private boolean[] source;
+	private Rule rule;
 
-	public Board(int width, int height) {
+	public Board(int width, int height, String ruleString) {
 		size = new Vector2i(width, height);
 		int cellsCount = width * height;
+		rule = new Rule(ruleString);
```

Comment:
Eventually the `Rule` should be changeable during runtime.
Follow Dependency Inversion and inject it rather than creating it here.